internal/complete/complete.go

```diff
@@ -139,10 +139,8 @@ func doComplete(compLine string, compWord string) {
 			return
 		}
 
-		// Append newline if it's missing. (https://github.com/antonmedv/fx/issues/303)
-		if input[len(input)-1] != '\n' {
-			input = append(input, '\n')
-		}
+		// Append newline to make sure that input is valid.
```

Comment:
=) comment is wrong. new line is needed to not glue with next peace of code. also `;` will work too